src/components/Book.js

```diff
@@ -1,20 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 function Book({ book, removeBook }) {
   return (
     <div>
       <div className="book-section" key={book.item_id}>
         <div className="book-innerSection">
           <span className="book-category">{book.category}</span>
-          <p className="book-title">{book.title}</p>
+          <h5 className="book-title">{book.title}</h5>
           <span className="book-author">{book.author}</span>
           <div className="btn-container">
```

Comment:
- Great job for improving the style based on previous reviwer seggetions., I think you missed one thing witch is to update the `nav items` `book` and `category`, so style them based on given design to have a better design as previous reviewer suggested 👍 
<img width="534" alt="2023-11-04_13-41" src="https://github.com/Obimbo07/book-library/assets/52010789/efee2021-b201-4944-adab-766595d1d737">

Reply:
Done